modules/support-gather-data.adoc

```diff
@@ -22,6 +22,11 @@ You can gather debugging information about your cluster by using the
 ----
 $ oc adm must-gather
 ----
++
+[NOTE]
+====
+If this command fails, for example if you cannot schedule a Pod on your cluster, then use the `oc adm inspect` command to gather information for particular resources. Contact Red Hat Support for the recommended resources to gather.
```

Comment:
@sferich888 Would you mind reviewing this statement to make sure you're okay with the wording about contacting support? 

Preview: https://osdocs-741--ocpdocs.netlify.com/openshift-enterprise/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data

Reply:
@deads2k are we ready to open `oc adm inspect` up to the world?